src/cortex_m4.rs

```diff
@@ -170,7 +170,7 @@ impl SecretKey {
     /// Internally, draws 256-bit `k` repeatedly, until signing succeeds.
     pub fn sign_prehashed(
         &self,
-        prehashed_message: &[u8],
+        prehashed_message: [u8; 32],
```

Comment:
Do you have strong opinions on `[u8; 32]` vs `&[u8; 32]` here?

Reply:
I changed that one to match `Scalar` in `p256`, the API changed and it now takes ownership instead of a reference: https://docs.rs/p256/latest/p256/struct.Scalar.html#method.from_be_bytes_reduced

I have no opinion, but if it is `&[u8; 32]` then a copy has to be made internally.